specification/columns/usagetype.md

```diff
@@ -0,0 +1,31 @@
+# Usage Type
+
+The Usage Type describes the type of service usage that is being billed for in a given charge. Cloud services have billing models in which charges are created based on more than one usage parameter. This field enables practitioners to differentiate between when charges for a given service are for different types of usage. For example, an object storage service may have charges for usage types of object storage, API requests, data transfer, encryption, object management, and more, and separate charges may appear in a FOCUS dataset for each of these.
```

Comment:
I believe this column should focus on the SKU (an underlying tracked and measured component of a resource or service) rather than the service.

I'm unsure whether it should provide only the core function or aspect of the SKU or include additional details as well. If we plan to introduce separate columns for SKU Size, SKU Region (potentially in a JSON format), etc., I would suggest focusing on the core function of the SKU for this column. 

That said, I understand that providers might initially map native columns that include more than just the core function, but I hope that over time this additional information will be parsed out and provided in separate columns. 
(Some practitioners and vendors already parse out this information, and it would be great if providers could do the same for us :) )

Potential mapping candidates:
- AWS: UsageType or [UsageTypeGroup](https://docs.aws.amazon.com/cost-management/latest/userguide/ce-filtering.html)
- Microsoft: Meter or MeterCategory?

Reply:
Since the column definition has changed, this comment is outdated, and I'm resolving it.